### ISSUE TITLE
compile fails on osx

### ISSUE THREAD

**Issue by jconley** [shundhammer/qdirstat#65]
`brew install -d -v qdirstat`
```
...
Undefined symbols for architecture x86_64:
  "_main", referenced from:
     implicit entry/start for main executable
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

was going to create a homebrew (http://brew.sh) formula for this. here's the formula-in-progress to test with...
```
class Qdirstat < Formula
  desc "QDirStat - Qt-based directory statistics"
  homepage "https://github.com/shundhammer/qdirstat"
  url "https://github.com/shundhammer/qdirstat/archive/1.4.tar.gz"
  sha256 "7f9a0a7304c55ebfad1e50c7747ba762b07068f48304b3d3919bdb65ee037999"
  head "https://github.com/shundhammer/qdirstat.git"

  depends_on "qt"

  def install
    # ENV.deparallelize  # if your formula fails when building in parallel
    system "qmake"
    system "make"
  end
end
```

**Comment by shundhammer** [comment]
See issue #14.

Please make sure you have a working Qt development environment and that you can build simple Qt examples.

**Comment by jconley** [comment]
@shundhammer qt environment seems fine. disclaimer: i'm not a qt/qmake expert. the app itself (under src) is building fine and runs. the problem seems to be that the .pro files for doc, doc/stats and scripts have a TARGET set to "app", which on macos is trying to compile a .app in each of those subfolders... hence why it's failing to find _main.
i'm not sure if TARGET should be set to something else... there are only 3 options in the documentation. i changed those 3 .pro files to use "TARGET = subdirs" and the build now works fine. thoughts?

**Comment by shundhammer** [comment]
Those "app" targets in those directories were kludges to make those directories build at all; qmake does have its limitations when it comes to things like that that are trivial in plain make. It's a bit over-eager to do its own thing everywhere, and it does not allow you to just add simple make rules as the autotools would.

That said, if this creates problems on your platform, maybe you might want to try to exclude those subdirs entirely from the build process in the project's toplevel .pro file; very likely installing those doc files to the typical directories for Linux systems won't do you any good on OS X anyway.

It might be possible to do that condition-based in the toplevel .pro file, i.e. exclude them only on OS X. If that works and you can come up with a pull request that doesn't break it on the other platforms, I will be more than happy to merge it.
